WasatchNET/Spectrometer.cs

```diff
@@ -2264,6 +2289,39 @@ internal bool sendCmd(Opcodes opcode, ushort wValue = 0, ushort wIndex = 0, byte
             return true;
         }
 
+        /// <summary>
+        /// send a single 2nd-tier control transfer command (response not checked)
+        /// </summary>
+        /// <param name="opcode">the desired command</param>
+        /// <param name="wIndex">an optional secondary argument used by some 2nd-tier commands</param>
+        /// <param name="buf">a data buffer used by some commands</param>
+        /// <returns>true on success, false on error</returns>
+        /// <todo>should support return code checking...most cmd opcodes return a success/failure byte</todo>
+        internal bool sendCmd2(Opcodes opcode, ushort wIndex = 0, byte[] buf = null)

```

Comment:
Do we not currently have a second-tier send?

Reply:
I truly don't know how we don't, but no I couldn't find one...